partials/tags.html

```diff
@@ -1,8 +1,14 @@
 {% macro tags(tagList) %}
+{% if tagList %}
 <div class="mt-4">
   {% for tag in tagList %}
-    <span class="bg-grey-dark font-normal text-sm no-underline text-white rounded lowercase mr-2 px-2 py-1" title="{{tag.description()}}">{{tag.name()}}</span>
+      {% if tag.externalDocs() %}
+        <span class="bg-grey-dark font-normal text-sm no-underline text-white rounded lowercase mr-2 px-2 py-1" title="{{tag.description()}}"><a href="{{tag.externalDocs().url()}}" target="_blank">{{tag.name()}}</a></span>
```

Comment:
Just a quick thing: this results in black over dark-grey background. The contrast is really low. Can we please choose another color for the text? Maybe just white? Also, the span tag here is unnecessary. Replace it with the `a` tag straight ahead.

```suggestion
        <a href="{{tag.externalDocs().url()}}" target="_blank" class="bg-grey-dark font-normal text-sm no-underline text-white rounded lowercase mr-2 px-2 py-1" title="{{tag.description()}}">{{tag.name()}}</a>
```

Reply:
I just realized that in this case you may want to leave the underline to indicate it's a link:

```suggestion
        <a href="{{tag.externalDocs().url()}}" target="_blank" class="bg-grey-dark font-normal text-sm text-white rounded lowercase mr-2 px-2 py-1" title="{{tag.description()}}">{{tag.name()}}</a>
```